scripts/01_various_recipe_fixes.zs

```diff
@@ -19,3 +22,6 @@ recipes.addShapeless(<thaumcraft:nugget:8> * 18, [zirconium_dust, copper_dust]);
 recipes.remove(<advancedrocketry:misc:1>, false);
 recipes.remove(<advancedrocketry:oxygenscrubber>, false);
 recipes.addShaped(<advancedrocketry:oxygenscrubber>, [[iron_bars, <libvulpes:productfan:6>, iron_bars], [iron_bars, <ore:blockMotor>, iron_bars], [iron_bars, carbon_plate, iron_bars]]);
+
+# Add recipe for saddle Aether added before removal
+recipes.addShaped(<minecraft:saddle>, [[null, null, null], [leather, leather, leather], [leather, string, leather];
```

Comment:
Looks like there's some syntax errors here. I think the following line would work:
```zs
recipes.addShaped(<minecraft:saddle>, [[null, null, null], [leather, leather, leather], [leather, string, leather]]);
```
Untested, of course ^^

Reply:
Made the suggested change. Look it over for any other issues.

Reply:
Let's break all the things!